lib/common/util.sh

```diff
@@ -4,88 +4,6 @@
 # HELPER PRIVATE FUNCTIONS #
 ############################
 
-# bulk index/unindex files using talend
-# $1 - directory to cd to before running harvester
-# $2 - base to index files with (prefix, such as IMOS/Argo)
-# $3 - file containing list of files to index
-# "$@" - extra parameters to $HARVESTER_TRIGGER
-_bulk_index_operation() {
-    local cd_to=$1; shift
-    local base=$1; shift
-    local file_list=$1; shift
-
-    test -z "$HARVESTER_TRIGGER" && log_info "Indexing disabled" && return 0
-
-    log_info "Bulk indexing/unindexing files from '$file_list'"
-
-    local tmp_harvester_output=`mktemp`
-    local log_file=`get_log_file $LOG_DIR $file_list`
-    (cd $cd_to && cat $file_list | $HARVESTER_TRIGGER --stdin -b $base "$@" >& $tmp_harvester_output)
-    local -i retval=$?
-
-    cat $tmp_harvester_output >> $log_file
-    rm -f $tmp_harvester_output
-    if [ $retval -ne 0 ]; then
-        # log to specific log file and not the main log file
-        log_error "Bulk indexing failed for '$file_list', verbose log saved at '$log_file'"
-    fi
-
-    return $retval
-}
-export -f _bulk_index_operation
-
-# bulk index files using talend
-# $1 - directory to cd to before running harvester
-# $2 - base to index files with (prefix, such as IMOS/Argo)
-# $3 - file containing list of files to index
-index_files_bulk() {
```

Comment:
This function, along with `unindex_files_bulk` could occasionally still be useful.
You can get by just calling `$HARVESTER_TRIGGER` directly, but it won't automatically save the log.

Reply:
I re-added them.

Reply:
:+1: Thanks.